plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/verifyEcdsaSecp256k1Signature/test-vector-16/test-vector-16.uplc.budget.expected

```diff
@@ -0,0 +1 @@
+evaluation failure
```

Comment:
Well, that's not exactly a budget.

Reply:
It's like that for all of the  tests that produce `error` ([here](https://github.com/IntersectMBO/plutus/blob/master/plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/divideInteger/divideInteger-zero/divideInteger-zero.uplc.budget.expected) for instance).  Maybe we should fix it?